.github/workflows/amoy_deb_profiles.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```

Comment:
Same as bor, why aren't we using a release versions for all the actions in the CI?

Reply:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions